frontend/public/components/replication-controller.jsx

```diff
@@ -60,7 +60,25 @@ const CancelAction = (kind, obj) => ({
   accessReview: asAccessReview(kind, obj, 'patch'),
 });
 
-const menuActions = [CancelAction, ...replicaSetMenuActions];
+const RollBackAction = (kind, obj) => ({
+  // t('public~Roll Back')
+  labelKey: 'public~Roll Back',
+  // disabled if the DC is not Active
+  isDisabled: _.includes(
```

Comment:
This should be disabled for replication controllers that aren't owned by a deployment config.

Reply:
nit: prefer native `Array.includes` to lodash